packages/perseus/src/widgets/__testdata__/randomizers.ts

```diff
@@ -0,0 +1,48 @@
+// Helpers to generate random test data for widgets
+
+// Returns a random letter inclusive of emoji and japanese characters
+export const randomLetter = (): string => {
+    const alphabet = "abcdefghijklmnopqrstuvwxyz".split("");
+    const emojis = ["🍇", "🍈", "🍌"];
+    const japaneseCharacters = ["ツ", "デ", "ト", "ド", "ナ", "ぁ", "あ"];
+    const characters = alphabet.concat(emojis).concat(japaneseCharacters);
+    const letter = randomInteger(0, characters.length - 1);
+    return characters[letter];
+};
+
+// Returns a random word 3-20 characters long composed of the random letter function
+export const randomWord = (): string => {
+    // words are 3 - 20 characters
+    const numLetters = randomInteger(3, 11);
+    return arrayOfLength(numLetters).map(randomLetter).join("");
+};
+
+// Returns a random sentence of a specified number of words, using the random word function
+export const randomSentence = (maxWords: number): string => {
+    const numWords = randomInteger(5, Math.max(maxWords, 5));
+    return arrayOfLength(numWords).map(randomWord).join(" ");
+};
+
+// generates a boolean. 50/50 unless rateTrue is specified
+export const randomBoolean = (rateTrue?: number): boolean => {
+    const threshold = rateTrue || 0.5;
+    return Math.random() < threshold;
+};
+
+// generates an integer inclusive of the specified min and max
+export const randomInteger = (min: number, max: number): number => {
+    return Math.floor(Math.random() * (max + 1 - min)) + min;
+};
+
+// A helper to create an array of a certain length.
+export const arrayOfLength = (length: number): number[] => {
```

Comment:
is there a better way to do this? I want range from python - but didn't name this range because it's not exactly the same